Xero.Api/IXeroApiSettings.cs

```diff
@@ -14,8 +14,6 @@ public interface IXeroApiSettings
 
         string SigningCertificatePassword { get; }
 
-        string AppType { get; }
-
-        bool IsPartnerApp { get; }
```

Comment:
These changes constitute a (minor) breaking change to the interface. Is there a reason to have both `AppType` and `IsPartnerApp`?

Reply:
Nope, no good reason that I can think of